includes/admin/helpers/class-rop-post-format-helper.php

```diff
@@ -141,6 +141,15 @@ public function build_content( $post_id ) {
 			$custom_messages = array_values( $custom_messages );
 			$random_index    = rand( 0, ( count( $custom_messages ) - 1 ) );
 			$share_content   = $custom_messages[ $random_index ]['rop_custom_description'];
+
+			if ( class_exists( 'Rop_Pro_Post_Format_Helper' ) ) {
+				$pro_format_helper = new Rop_Pro_Post_Format_Helper;
+			}
+
+			if ( ! empty( $pro_format_helper ) ) {
```

Comment:
@UVLabs you should use isset instead of empty as this will trigger undefined variable notice. 
Or you set the variable `$pro_format_helper` first as null.

Reply:
@selul thanks ill use isset